registration/registration-client/src/main/java/io/mosip/registration/controller/BaseController.java

```diff
@@ -601,6 +602,19 @@ public void goToHomePage() {
 		}
 	}
 	
+	/**
+	 * Opens the home page screen.
+	 */
+	public void goToNewRegistration() {
+		goToHomePage();
+		if (packetHandlerController.getRegistrationGridPane() != null && packetHandlerController.getRegistrationGridPane().getChildren().size() > 0) {
+			Event.fireEvent(packetHandlerController.getRegistrationGridPane().getChildren().get(0), new MouseEvent(MouseEvent.MOUSE_CLICKED, 0,

```

Comment:
I think it will be incorrect to always assume that the first child is a "New Registration". 
Why should we do this?

Reply:
We have "New Registration" button at Acknowledgement screen right, It is right now, taking to home screen. To open New Registration, accessing the gridpane's first child assuming it as "New Registration". Couldn't find another alternative to navigate to New Registration directly.. any suggestions Anusha?

Reply:
It is handled by assigning the registration-type as the id for each child in the registration grid-pane.